qadi.py

```
@@ -125,12 +126,71 @@
+    print("OUTPUT:")
+    print("  • Key question identified")
+    print("  • Creative hypothesis generated") 
+    print("  • Logical deduction applied")
+    print("  • 3 practical actionable answers")
+    print("  • Performance metrics (time, cost, model)")
+    print()
+    print("VS MULTI-AGENT VERSION:")
+    print("  • qadi.py: Fast single call, good for quick questions")
+    print("  • qadi_simple_multi.py: Deeper multi-perspective analysis")
+
+def show_version():
```

Ruff (0.12.2)

Missing return type annotation for public function `show_version`

Add return type annotation: `None`

(ANN201)